src/Storage/Data/IPackagingDataProvider.cs

```diff
@@ -1,17 +1,20 @@
 ﻿using System;
 using System.Collections.Generic;
+using SenseNet.Tools;
 
 namespace SenseNet.ContentRepository.Storage.Data
 {
-    public interface IDataProcedureFactory
+    public static class Extensions
```

Comment:
Maybe a more specific class name?
```suggestion
    public static class PackagingExtensions
```